salt/runners/fileserver.py

```diff
@@ -322,7 +322,7 @@ def empty_dir_list(saltenv="base", backend=None):
     return fileserver.file_list_emptydirs(load=load)
 
 
-def update(backend=None):
+def update(backend=None, **kwargs):
```

Comment:
Please update this signature to include the specific arguments that should be passed